### ISSUE TITLE
New Game Critical Error

### ISSUE THREAD

**Issue by samueljlieber** [samueljlieber/wefrolf#10]
# Error:

When creating a new game this error is showing on redirect to hole 1.

```
Error: Abort fetching component for route: "/game/[gid]/[cid]/[hid]"
```

**Comment by samueljlieber** [comment]
![image](https://user-images.githubusercontent.com/36018073/225480353-2fab9860-f6e8-4806-a6db-a1258087c118.png)

![image](https://user-images.githubusercontent.com/36018073/225480385-600f6c00-b3f1-48d8-b676-0ddc5c11bceb.jpeg)

![image](https://user-images.githubusercontent.com/36018073/225480402-2299adb2-d049-4e56-8a0e-3dd4327b4a66.jpeg)

This error is happening on mobile specifically, and only when using a NON WeFrolf course.

**Comment by samueljlieber** [comment]
Issue fixed in 30d77319963c1d5e6fd8a55000134a7695ea8e59
